### ISSUE TITLE
Not able to select already existing tag

### ISSUE THREAD

**Issue by thelazyoxymoron** [go-shiori/shiori#386]
I'm not able to select any tags from the matched list. Using the Arm64 docker image built using the latest codebase.

**Comment by fmartingr** [comment]
Where do you have this problem? In the search, in the article list, in the tag list, in the "add bookmark" dialog?

**Comment by thelazyoxymoron** [comment]
I'm facing this in the add new bookmark screen, as well as when I try to edit an existing bookmark. DB is Postgresql.

**Comment by fmartingr** [comment]
Just to confirm, the problem is that you don't see the tags input like this:

![2022-03-01_12-32-36](https://user-images.githubusercontent.com/812088/156170000-13e7b4af-184d-42a6-bf26-3dbf1b9af3d4.png)

Which browser are you suing?

**Comment by thelazyoxymoron** [comment]
> Just to confirm, the problem is that you don't see the tags input like this:

I see the tags input getting suggested, but not able to select from the suggested list.

> Which browser are you suing?

Firefox v97

**Comment by fmartingr** [comment]
I've just tried this on Firefox 97.0.2 @ macos 12.1 and it's working:

https://user-images.githubusercontent.com/812088/156902737-53594b4c-f055-4fc4-8e8f-57195ecf2ed7.mov

I put a few letters and press enter on the suggestion.

**Comment by mrex** [comment]
After finding this bug report, I guess I finally understand how tag suggestions are supposed to work. It looks like Shiori's suggestion shows only one of the possible tags, and only when the text entered does not match the suggestion anymore, another possible tag is shown.

That's not intuitive at all. Usually, when you enter a tag the suggestions show all the tags matching the substring typed so far, with a way to select one of the (possibly multiple) suggested tags, either by navigating with the keyboard or by clicking on it.

**Comment by fmartingr** [comment]
Moving this from bug to enhancement, and while agree that the current selector is not optimal I can't consider this a "bug" since it's working as expected. At least until we improve it :)

@thelazyoxymoron if you are experiencing an actual bug please report back so we can take a look at it (and proper triage a new issue for the feature request).

**Comment by thelazyoxymoron** [comment]
Apologies for not providing enough information earlier. I did some more testing and looks like this is only happening with Postgres DB backend and only when I try to add a tag in the following window:

![image](https://user-images.githubusercontent.com/9106868/193747155-a6e60378-9166-4634-832f-dfd32328c845.png)

I did not observe this when trying to edit an existing bookmark and add tags. The auto-suggest + enter was working there. Only observing when adding tags in the "add new bookmar" screen.

This is related to #215.

**Comment by fmartingr** [comment]
I've been testing this locally, and after #484 fix for PSQL it now works but there's a caveheat: the frontend loads tags on page load, so if you add a new bookmark with a new tag and then add another bookmark that new tag wont autocomplete unless the page is reloaded.

**Comment by v-marinkov** [comment]
I also confirm I can't click on the suggested tags (Docker, sqlite3, Firefox, Windows). It would be great to have the tags suggestion in the browser extension too (https://github.com/go-shiori/shiori-web-ext/issues/64)
